Examples/SnowplowSwiftCarthageDemo/Cartfile

```diff
@@ -0,0 +1 @@
+git "file:///Users/alexbenini/dev/snowplow/snowplow-objc-tracker" "460-update_swift_demo_app_to_1_1_4"
```

Comment:
This will need altering to Github master before merge, or if you are intending users to run generateCartfile.sh before running the application then I'd probably delete this file and .gitignore it. My issue is that even what generateCartfile.sh does isn't what you would want to do in your own application so it isn't an ideal example.
Maybe include a cartfile that points to github but leave generateCartfile that can override it to the current branch for easy testing?